misc/python/materialize/version_list.py

```diff
@@ -68,7 +68,21 @@
 }
 
 """
-See: #_MIN_ANCESTOR_MZ_VERSION_PER_COMMIT_TO_ACCOUNT_FOR_PERFORMANCE_AND_SCALABILITY_REGRESSIONS
+See: #_MIN_ANCESTOR_MZ_VERSION_PER_COMMIT_TO_ACCOUNT_FOR_PERFORMANCE_REGRESSIONS
+"""
+_MIN_ANCESTOR_MZ_VERSION_PER_COMMIT_TO_ACCOUNT_FOR_SCALABILITY_REGRESSIONS: dict[
+    str, MzVersion
+] = {
+    # insert newer commits at the top
```

Comment:
Why? It's a dict, so order doesn't matter.

Reply:
Dictionaries are insertion ordered since Python 3.7 (https://stackoverflow.com/a/39980744/584532). I iterate over the entries.

Reply:
Oh wow, wasn't aware of this change.